api/helpers.go

```diff
@@ -64,6 +54,52 @@ func (a *API) buildLoginResponse(id string) (*LoginResponse, error) {
 	return &lr, nil
 }
 
+// sendUserCode method allows to send a code to the user via email or SMS. It
+// generates a verification code and stores it in the database associated to
+// the user email. If the mail service is available, it sends the verification
+// code via email. If the SMS service is available, it sends the verification
+// code via SMS. The code is generated associated a the type of code received,
+// that can be either a verification code or a password reset code. Other types
+// of codes can be added in the future. If neither the mail service nor the SMS
+// service are available, the verification code will be empty but stored in the
+// database to mock the verification process in any case.
+func (a *API) sendUserCode(ctx context.Context, user *db.User, codeType db.CodeType) error {
```

Comment:
@lucasmenendez it seems that this function is used only by the `api/users.go` module. MAybe it could be moved there?

Reply:
Yes, you're right.